ios/FluentUI/Core/Theme/Tokens/GradientInfo.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import CoreGraphics
+import Foundation
+import SwiftUI
+
+/// Represents a gradient as used by FluentUI.
+@objc public class GradientInfo: NSObject {
+    /// Initializes a gradient to be used in Fluent.
+    ///
+    /// - Parameters:
+    ///   - colors: The color of the shadow for shadow 1.
+    ///   - startPoint: The blur of the shadow for shadow 1.
+    ///   - endPoint: The horizontal offset of the shadow for shadow 1.
```

Comment:
This comment looks suspiciously `ShadowInfo`y

Reply:
Oh no, you've discovered my secret copy-and-paste approach to comment blocks! (will fix)